src/libsync/logger.cpp

```diff
@@ -118,7 +118,8 @@ bool Logger::isLoggingToFile() const
 
 void Logger::doLog(QtMsgType type, const QMessageLogContext &ctx, const QString &message)
 {
-    const QString msg = qFormatLogMessage(type, ctx, message);
+    static long long int linesCounter = 0;
+    const auto &msg = qFormatLogMessage(type, ctx, message);
```

Comment:
@mgallien does using a reference help improve performance?

Reply:
we do construct one less `QString` instance
even if there is a reference counter, why do something that is not needed in a performance sensitive function ?